src/type/__tests__/enumType.js

```diff
@@ -30,6 +30,13 @@ describe('Type System: Enum Values', () => {
     }
   });
 
+  var FavoriteType = new GraphQLObjectType({
```

Comment:
This type seems unnecessary for the test. Should `Mutation.favoriteEnum`'s type just be `ColorType`?